include/glow/Backends/Backend.h

```diff
@@ -87,6 +87,20 @@ class Backend {
 /// Create a backend of kind \p kind, to run the IR function \p M.
 Backend *createBackend(BackendKind backendKind, IRFunction *M);
 
+/// Create a new instance of the interpreter backend.
+Backend *createInterpreter(IRFunction *F);
```

Comment:
Since these are essentially forward decls, I wonder if we could move them into Backends.cpp?  My motivation being limiting the number of files that need to perform #ifdefs to get the right bits :)

Reply:
This was my initial idea as well. But then I realized that we probably need these forward decls to be included into cpp files implementing these functions. This is the only reason I put them into the include file.